hangman.js

```diff
@@ -0,0 +1,132 @@
+    
+    
+    let youDoWantTo = document.querySelector("#yes")
+    youDoWantTo.addEventListener("click", () => {
+        event.preventDefault()
+        document.getElementById("chiptune").play()        
+        document.getElementById("page1").style.display = "none"
+        document.getElementById("page2").style.display = "grid"
+    })
+
+    let youDoNotWantTo = document.querySelector("#no")
+    youDoNotWantTo.addEventListener("click", () => {
+        window.location.href = "http://BigMacKrell.com"
+    })
+
+    let turnToPage3 = document.getElementById("press-to-talk1")
+    turnToPage3.addEventListener("click", () => {
+        event.preventDefault()
+        document.getElementById("page2").style.display = "none"
+        document.getElementById("page3").style.display = "grid"
+    })
+
+    let turnToPage4 = document.getElementById("press-to-talk2")
+    turnToPage4.addEventListener("click", () => {
+        event.preventDefault()
+        document.getElementById("page3").style.display = "none"
+        document.getElementById("page4").style.display = "grid"
+    })
+
+    let turnToPage5 = document.getElementById("press-to-talk3")
+    turnToPage5.addEventListener("click", () => {
+        event.preventDefault()
+        document.getElementById("page4").style.display = "none"
+        document.getElementById("page5").style.display = "grid"
+    })
+
+    let arr = ["stan", "grunkle", "dipper", "mabel", "soos", "mcgucket", "mcskermish", "greasys", "susan", "candy", "greta", "stanford", "stanley", "dimensional", "caesar", "cipher", "blendin", "blandin", "journal", "cider", "summerween", "jackalope", "bigfoot", "alien", "gobblewonker", "mystery", "shack", "gravity", "triangle", "pacifica", "waddles", "mackerel", "swine", "bottomless"]
+
+    let threeLettersBack = "DEFGHIJKLMNOPQRSTUVWXYZABC"
+
+    let decodedAlpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+    let randomWord = arr[Math.floor(Math.random() * arr.length)]
+
+    console.log(`CHEAT CODE ACCEPTED: \nTHE ANSWER IS ${randomWord}`)
+
+    let splitRandomWord = randomWord.split("")
+
+    let dashHolder = []
+
+    for (let i = 0; i < splitRandomWord.length; i++) {
+    dashHolder.push("_ ");
+    }
+
+    let caesarHolder = ""
+
+    for (let i = 0; i < splitRandomWord.length; i++) {
+        for (let j = 0; j < decodedAlpha.length; j++) {
+            if (splitRandomWord[i].toUpperCase() === decodedAlpha[j]) {
+                caesarHolder += threeLettersBack[j]
+            }
+        }
+    }
+    
+    let letterCatcher = []
+    console.log(letterCatcher)
+
+    document.getElementById("encoded-word").innerHTML = caesarHolder
+    document.getElementById("dash-holder").innerHTML = dashHolder.join("");
+    document.getElementById("correct-answer").innerHTML = `THE CORRECT ANSWER WAS: ${randomWord.toUpperCase()}`
+    
+    let wrongAnswers = 0
+    
+    
+    let form = document.querySelector("#form4")
+    
+    form.addEventListener("submit", (event) => {
```

Comment:
Even though this code works, you could also use functions to separate if statements into multiple functions.